apiary.apib

```diff
@@ -4455,3 +4455,156 @@ Gets stats for a given Run Id. Provides the count of imported/exported tables an
                     }
                 }
             }
+# Group Triggers
+
+###Create [POST /v2/storage/triggers/]
+Created a trigger listeng on set of tables
+
++ Attributes
+    + runWithTokenId (required, number) - Id of token used for running configured component
+    + componentType: `orchestration` (required, enum[string]) - For now we support only 'orchestration'
+    + configurationId (required, number) - Id of component configuration
+    + coolDownPeriod (required, number) - Minimal cool down period before firing action again in minutes (min is 5 minutes)
+    + tabledIds (required, object) - Array of tables ids
+        + tableId (string) - Id of table (like "in.c-test.watched")
++ Request
+
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+    + Body
+
+            runWithTokenId=123&componentType=orchestration&configurationId=123&coolDownPeriod=20&tableIds[0]=in.c-test.watched-1&tableIds[1]=in.c-prod.watched-5
+
++ Response 201 (application/json)
+
+        {
+            "id": 3,
+            "runWithTokenId": 123,
+            "projectId": 234,
+            "componentType": "orchestration",
```

Comment:
dal bych jenom `component` ať je to konzistetní se zbytkem https://keboola.docs.apiary.io/#reference/workspaces/workspaces-collection/list-workspaces .

Reply:
Počítám s tím že pod tím compoent fakt čekáme její id (`orchestrator`, `keboola.ex-aws-s3`, `keboola.wr-aws-s3`) a ne typ (writer,extractor,application, other)

Reply:
přejmenoval jsem to